scripts/multinode/configure-sec.sh

```diff
@@ -40,6 +40,9 @@ function configure_microshift() {
     sleep 5
     sudo systemctl start microshift-ovs-init.service
 
+    # Add dummy IP to br-ex
+    sudo ip addr add 10.44.0.0/32 dev br-ex
```

Comment:
Why are we choosing the IP from 10.44.0.0 subnet? Is it related to some default configuration setting?

Reply:
We use this IP in the single node cluster as the default KAS advertise address. We can use any IP as long as it doesn't overlap with any other internal/external IP ranges.